pkg/pub_package_reader/lib/pub_package_reader.dart

```diff
@@ -307,6 +308,21 @@ Iterable<ArchiveIssue> validatePackageVersion(Version? version) sync* {
   }
 }
 
+/// Checks if the [text] has any weird characters like in
+/// https://en.wikipedia.org/wiki/Zalgo_text
+Iterable<ArchiveIssue> validateZalgo(String field, String? text) sync* {
+  if (text == null) return;
+  bool isDiacritic(int code) => code >= 0x0300 && code <= 0x036F;
```

Comment:
What about "Combining Diacritical Marks Extended", "Combining Diacritical Marks , Supplement" , " for Symbols":

https://en.wikipedia.org/wiki/Combining_character (see "Unicode ranges")

Reply:
Good point, extended the ranges.